packages/grpc-native-core/src/server.js

```diff
@@ -868,7 +868,7 @@ var defaultHandler = {
  *     names to method implementation for the provided service.
  */
 Server.prototype.addService = function(service, implementation) {
-  if (!_.isObject(service) || !_.isObject(implementation)) {
+  if (typeof service !== 'object' || typeof implementation !== 'object') {
```

Comment:
This should probably also check for `null` explicitly since `typeof null === 'object'`.

Reply:
I agree. In addition, the lodash function also returns `true` for values with a `typeof` that outputs `"function"`. That may be undesirable, though.